libs/product/src/drivers/magento/queries/fragments/product.ts

```diff
@@ -0,0 +1,35 @@
+import gql from 'graphql-tag';
+
+export const magentoProductFragment = gql`
+  fragment product on ProductInterface {
```

Comment:
I think this maybe should be a concrete type that the fragment operates on. Not totally sure though and we can come back to perfecting the fragments later. I doubt it will actually cause any problems.

Reply:
This needs to be "ProductInterface" or the magento call will fail

Reply:
hm maybe I'm wrong then. In that case the cart fragments might need changing.